aip/general/0001.md

```diff
@@ -89,7 +89,6 @@ The list of AIP editors is currently:
 - Hong Zhang ([@wora][])
 - JJ Geewax ([@jgeewax][])
 - Jon Skeet ([@jskeet][])
-- Luke Sneeringer ([@lukesneeringer][])
```

Comment:
Add yourself instead? You don't need to choose to use that approval status until you're comfortable with it...